test/sql-tap/gh-6024-funcs-return-bin.test.lua

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env tarantool
+local build_path = os.getenv("BUILDDIR")
+package.cpath = build_path..'/test/sql-tap/?.so;'..build_path..'/test/sql-tap/?.dylib;'..package.cpath
+
+local test = require("sqltester")
+test:plan(1)
+
+box.schema.func.create("gh-6024-funcs-return-bin.ret_bin", {
+    language = "C",
+    param_list = { "varbinary" },
+    returns = "varbinary",
+    exports = { "LUA", "SQL" },
```

Comment:
Why export lang is LUA? And do you really need is_deterministic flag?

Reply:
Fixed.